code/style.css

```diff
@@ -1,4 +1,192 @@
 html {
-  background: #0025ff;
-  color: #fff;
+background: white;
+color: #fff;
 }
+
+body {
+margin: 0%;
+font-family: 'Roboto', sans-serif;
+}
+
+header {
+background-image: url("https://images.unsplash.com/photo-1629573010718-5a4a1aea6c23?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1170&q=80");
+padding: 275px 0;
+background-size: cover;
+background-position: center;
+background-repeat: no-repeat center;
+}
+
+.navigation-bar {
+height: 100px;
+background-color: black;
+display: flex;
+}
+
+.matchsticksnews-logo {
+width: 100px;
+height: 100px;
+filter: contrast(130%);
+}
+
+.site-name{
+font-size: 30px;
+}
+
+@media (min-width: 668px) {
+h1 {
+color:orangered;  
+  }
+}
+
+.navigation-bar a {
+display: none;  
+}
+
+@media (min-width: 668px) {
+.navigation-bar a {
+padding: 35px;
+text-decoration: none;
+color: white; 
+display: flex;
+  }  
+}
+
+@media (min-width: 668px) {
+.navigation-bar {
+justify-content: space-between;
+  }  
+}
+
+.icon-box {
+display: none;  
+}
+
+@media (min-width: 668px) {
+.icon-box {
+display: flex;
+  }
+}
+
+@media (min-width: 668px) {
+.icon-box a {
+display: flex;        
+  } 
+}
+ 
+@media (min-width: 668px) {
+.icon1 {
+background-color: white;
+  }
+}
+  
+@media (min-width: 668px) {
+.icon2 {
+background-color: white;
+  }
+}
+  
+@media (min-width: 668px) {
+.icon3 {
+background-color: white;
+  }
+}
+  
+.big-news {
+display: grid;
+grid-template-columns: 1fr;
+margin-top: 25px;
+margin-left: 25px;
+margin-right: 25px;
+margin-bottom: 50px;
+}
+
+.big-news-article {
+color: black; 
+}
+
+.big-news img{
+width: 500px;
+margin-top: 25px; 
+}
+
+.big-news a {
+text-decoration: none;
+color: orangered;
+}
+
+@media (min-width: 1025px) {
+.big-news a:hover {
+background-color: black;
+  }  
+}
+
+.hr {
+border-top: 5px solid;  
+color: grey;
+margin-left: 0%;
+margin-right:0%;
+}
+
+@media (min-width:668px) {
+.big-news {
+display:grid;
+grid-template-columns: repeat(2, 1fr); 
+grid-gap: 15px; 
+  }  
+}
+
+.other-news {
+display:grid;
+grid-template-columns: 1fr; 
+margin-top: 50px;
+margin-left: 25px;
+margin-right: 25px;
+margin-bottom: 50px;
+}
+
+.other-news img {
+width: 300px;  
```

Comment:
Suggestion to set the img width to 100% to make them fit better within the border